docs/quick_start/viewing.rst

```diff
@@ -1,17 +1,26 @@
 .. Images
-.. |viewing_data| image:: ../images/viewing_data.JPG
+.. |viewing_data| image:: ../images/viewing_data.jpg
```

Comment:
Better to take screenshot without top navbar as it has fledge Logo. With white labelled it becomes an issue with Product